vnext/package.json

```diff
@@ -69,5 +69,56 @@
       "minor",
       "patch"
     ]
-  }
+  },
+  "files": [
+    "/codegen",
```

Comment:
>codegen [](start = 6, length = 7)

this is produced from just-task.js on build, but it seems to be used from cpp projects. 

I'm not sure if this should move to msbuild to be part of the regular compile, so you can pull and rebuild without having to run `yarn build` and get F5 support when chancing the .js file. 

Because i doubt the metro bunder will take care o fthis on the fly..

Reply:
I think this one is reasonable to have in the JS build for now, but to my understanding the codegen stuff is still somewhat in prototype phase. Right now we only run it for native module specs built into React Native, so we would need to run the JS build anyway if we changed the underlying RN package.